### PR TITLE
Update diffusion.ipynb to addess the modifications in the Python API in STEPS 3.5.0

### PR DESCRIPTION
**Pull request by cattabiani**
Fix issue  #22.

### CODE REVIEW

File: user_manual/source/diffusion.ipynb [hunk 1]
```diff
@@ -475,7 +475,7 @@
     "    tidx = mesh.findTetByPoint([xcrd, ycrd, zcrd])\n",
```
Comment (tristan0x) [inline user_manual/source/diffusion.ipynb:475]: <p>This change is not necessary. Can you please remove it from the PR?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CNS-OIST/STEPS_Example/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Comment (tristan0x) [inline user_manual/source/diffusion.ipynb:475]: <p>This change is not necessary. Can you please remove it from the PR?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CNS-OIST/STEPS_Example/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

**Review by tristan0x — APPROVED** [review]
LGTM!

**Review by tristan0x — CHANGES_REQUESTED** [review]
Please fix the typo in your commit message.

**Comment by cattabiani** [comment]
> Please fix the typo in your commit message.

I am blind. Thx

**Review by tristan0x — APPROVED** [review]
LGTM!

**Pull request merged**